pom.xml

```diff
@@ -558,6 +561,7 @@
                         <groupId>org.apache.maven.plugins</groupId>
                         <artifactId>maven-surefire-plugin</artifactId>
                         <configuration>
+                            <useModulePath>false</useModulePath>
```

Comment:
I do not think this line is required given that there is `<useModulePath>false</useModulePath>` in the pluginManagement section.